examples/uplay.c

```diff
@@ -311,10 +322,14 @@ static int catch_video(struct uprobe *uprobe, struct upipe *upipe,
     upipe_schedule = upipe_void_chain_output(avcdec, subpic_schedule_mgr,
             uprobe_pfx_alloc(uprobe_use(uprobe), UPROBE_LOG_VERBOSE, "subpic schedule"));
     upipe_mgr_release(subpic_schedule_mgr);
-    upipe_release(upipe_schedule);
+
+    struct upipe_mgr *probe_uref_mgr = upipe_probe_uref_mgr_alloc();
+    struct upipe *upipe_probe_uref = upipe_void_chain_output(upipe_schedule, probe_uref_mgr,
+            uprobe_pfx_alloc(uprobe_use(uprobe), UPROBE_LOG_VERBOSE, "video probe_uref"));
```

Comment:
I'd prefer if you used a specific probe for each pipe, because in the log you will see chains of uprobe_pfx.